src/staking-router/staking-router.service.ts

```diff
@@ -13,6 +15,77 @@ export class StakingRouterService {
     protected readonly keysApiService: KeysApiService,
   ) {}
 
+  public async getVettedAndUnusedKeys() {
+    // TODO: add cache by modules nonce
+    const operatorsByModules =
+      await this.keysApiService.getOperatorListWithModule();
+    const operatorsBlockHash =
+      operatorsByModules.meta.elBlockSnapshot.blockHash;
+    const operatorsBlockNumber =
+      operatorsByModules.meta.elBlockSnapshot.blockNumber;
+
+    const unusedKeys = await this.keysApiService.getUnusedKeys();
+    const keysBlockHash = unusedKeys.meta.elBlockSnapshot.blockHash;
+    if (keysBlockHash != operatorsBlockHash) {
```

Comment:
it's always a good idea to use a strict comparison `!==`

Reply:
fixed, thank you